src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -18,14 +18,15 @@ def id(self) -> int:
 
 class TransactionalEmail(Enum):
     EMAIL_CONFIRMATION = Template(id_prod=201, id_not_prod=15, tags=["jeunes_confirmation_mail"])
-    EMAIL_ADDRESS_CHANGE_REQUEST = Template(id_prod=142, id_not_prod=17, tags=["changement_email_demande"])
-    EMAIL_ADDRESS_CHANGE_CONFIRMATION = Template(id_prod=253, id_not_prod=18, tags=["changement_email_confirmation"])
+    CHANGE_EMAIL_ADDRESS_REQUEST = Template(id_prod=142, id_not_prod=17, tags=["changement_email_demande"])
+    CHANGE_EMAIL_ADDRESS_CONFIRMATION = Template(id_prod=253, id_not_prod=18, tags=["changement_email_confirmation"])
```

Comment:
(question de goût) Je crois que j'avais déjà commenté là-dessus, mais quitte à renommer, autant supprimer la partie `ADDRESS_` : 

```suggestion
    EMAIL_CHANGE_REQUEST = Template(id_prod=142, id_not_prod=17, tags=["changement_email_demande"])
    EMAIL_CHANGE_CONFIRMATION = Template(id_prod=253, id_not_prod=18, tags=["changement_email_confirmation"])
```

C'est plus court et plus clair. Enfin je trouve. :)

Reply:
Ok